pkgs/development/python-modules/docling-parse/default.nix

```diff
@@ -0,0 +1,80 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  cxxopts,
+  poetry-core,
+  pybind11,
+  tabulate,
+  zlib,
+  nlohmann_json,
+  utf8cpp,
+  libjpeg,
+  qpdf,
+  loguru-cpp,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "docling-parse";
+  version = "2.0.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "DS4SD";
+    repo = "docling-parse";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-unXGmMp5xyRCqSoFmqcQAZOBzpE0EzgEEBIfZUHhRcQ=";
+  };
+
+  dontUseCmakeConfigure = true;
+
+  build-system = [
+    cmake
+    poetry-core
+    pybind11
+    pkg-config
```

Comment:
cmake and pkg-config should still go to nativeBuildInputs. Not sure about pybind11 but if it is taken from PATH than that one, too.

Reply:
How can I check `pybind11` ?

Reply:
Maybe, try to put it in `buildInputs`. If it's not working, then have it in `nativeBuildInputs`.
Anyway, I am pretty sure that it doesn't belong in `build-system`.

Reply:
It works when it is in `buildInputs`. I guess it can stay there?

Reply:
Yes, I agree.